x/evm/keeper/state_transition.go

```diff
@@ -429,6 +434,7 @@ func (k *Keeper) ApplyMessageWithConfig(
 		// Ethermint original code:
 		// stateDB.SetNonce(sender.Address(), msg.Nonce+1)
 		sgxRPCClient.StateDBSetNonce(StateDBSetNonceArgs{
+			EvmId:  evmId,
```

Comment:
In this call, let's also check the error:

```
err := sgxRPCClient.StateDBSetNonce(...
```

Reply:
Agree, done.